bot/exts/evergreen/snakes/__init__.py

```diff
@@ -1,12 +1,11 @@
 import logging
 
-from discord.ext import commands
-
+from bot.bot import Bot
 from bot.exts.evergreen.snakes._snakes_cog import Snakes
 
 log = logging.getLogger(__name__)
 
 
-def setup(bot: commands.Bot) -> None:
+def setup(bot: Bot) -> None:
```

Comment:
This docstring hasn't been updated.

Reply:
Resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/780aaeb5c272f7a9fd9fe9e8cdc1c041beb90a25.